plugin/file/zone.go

```diff
@@ -56,6 +56,16 @@ func NewZone(name, file string) *Zone {
 	return z
 }
 
+// GetPath returns the currently known file path
+func (z *Zone) GetPath() string {
+	return z.file
```

Comment:
If we export File we don't need these getter and setters

Reply:
+1 to avoid getter and setters.

Reply:
Yep, makes sense, I just wasn't sure how you guys felt about exporting that property. Will revise.